portal/client/src/components/ListView/ListView.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const ListView = ({studyList = [], handleClick}) => {
+    return (
+        <div className="col-4">
+            <div className="list-group">
+                { studyList.map((data, index) => {
+                    if(data) {
```

Comment:
Why dont we filter out data in the parent component before passing it to the this child component?
Just do this in parent before passing `studyList` props `studyList= studyList.filter(data => data)` , this will clear things out and you dont need to write this `if` statements